MM-22013 - Allow users to specify default download locations (#4569)
mattermost/docs: source/help/apps/desktop-guide.rst

To add a new server to your desktop app environment:

1. Select the **+** button in the desktop window bar at the top of the screen.
2. In the **Name** field, enter the name that you want for the tab.
3. In the **URL** field, enter the complete URL of the server that you want to connect to. URLs must begin with either ``http://`` or ``https://``.
4. Select **Add**.

Editing Servers
~~~~~~~~~~~~~~~

To edit a server in your desktop app environment:

> 1. On Windows, go to **... > File > Settings**. On Mac or Linux, go to **Mattermost > Preferences**.

Reviewer: ```suggestion
1. On Windows, go to **... > File > Settings**. On macOS or Linux, go to **Mattermost > Preferences**.
```

2. Next to the server you want to update, select **Edit**.
3. Edit **Name** and/or **URL**.
4. Select **Save**.

Removing Servers
~~~~~~~~~~~~~~~~

To remove a server from your desktop app environment:

> 1. On Windows, go to **... > File > Settings**. On Mac or Linux, go to **Mattermost > Preferences**.

Reviewer: ```suggestion
1. On Windows, go to **... > File > Settings**. On macOS or Linux, go to **Mattermost > Preferences**.
```

2. Next to the server or team that you want to remove, select **Remove**.

> 3. select **Remove**.

Reviewer: ```suggestion
3. Select **Remove**.
```

App Options
-----------

In addition to `Mattermost Account Settings <https://docs.mattermost.com/help/settings/account-settings.html>`__, the Desktop App provides additional options to customize your experience.

> The options appear on the Settings page, available on Windows from **... > File > Settings**, or on Mac or Linux from **Mattermost > Preferences**.

Reviewer: ```suggestion
The options appear on the Settings page, available on Windows from **... > File > Settings**, and on macOS or Linux from **Mattermost > Preferences**.
```

Use GPU hardware acceleration
    When enabled, the Mattermost user interface is rendered more efficiently but can lead to decreased stability for some systems. This setting is enabled by default. You must restart the app to change this setting.
    
Download Location
~~~~~~~~~~~~~~~~~

You can specify where files downloaded from the Mattermost desktop app are saved on your local machine. By default, files are saved to the ``Downloads`` folder.

> 1. On Windows, go to **... > File > Settings**. On Mac or Linux, go to **Mattermost > Preferences**.

Reviewer: ```suggestion
1. On Windows, go to **... > File > Settings**. On macOS or Linux, go to **Mattermost > Preferences**.
```
